### DISCUSSION TITLE
Use anyio to replace asyncio

### DISCUSSION THREAD

**Discussion by abersheeran** [abersheeran/a2wsgi#19] [Ideas]
https://github.com/agronholm/anyio

This will support both Trio and asyncio.

**Comment by provinzkraut** [comment]
Hello there, we're looking into using this library in [starlite](https://github.com/starlite-api/starlite), which would require `anyio` support. Are there currently plans to implement it? If not, would contributions adding `anyio` be welcome?

**Reply by abersheeran** [reply]
Contributions are welcome.
